stories/Datatable.stories.svelte

```diff
@@ -18,11 +18,12 @@ let numberOfCheckboxes = 0
 onMount(() =>
   setTimeout(() => {
     loaded = true
-  }, 5000)
+  }, 5000),
```

Comment:
Why the comma? Since it isn't in a list, that seemed a little unexpected (though I would guess it doesn't matter).

Reply:
prettier added that.  Not sure why.